examples/lambo.html

```diff
@@ -29,22 +29,61 @@
     text-align: center;
   }
 
+  #fs {
+    position: absolute;
+    top: 0px;
+    left: 0px;
+  }
+
   </style>
+  <script>
+  var vrDevices = {};
+
+  function start() {
+    var fsButton = document.querySelector('#fs');
+    var container = document.querySelector('#container');
+
+    fsButton.addEventListener('click', function() {
+      container.mozRequestFullScreen({ vrDisplay: vrDevices.headset });
+    })
+
+    window.addEventListener('resize', function(e) {
+      console.log('window Height = container Height', window.innerHeight == container.offsetHeight)
+    })
+
+  }
+
+  navigator.getVRDevices().then(function(devices) {
+    vrDevices.headset = devices[0];
+    vrDevices.position = devices[1];
+    
+    start();
+  });
+  </script>
```

Comment:
Ignore this stuff.  it was just meant to isolate some of the fullscreen requests happening in the WC